src/server/scripts/Outland/HellfireCitadel/MagtheridonsLair/boss_magtheridon.cpp

```diff
@@ -175,29 +175,33 @@ struct boss_magtheridon : public BossAI
             me->SetOrientation(me->GetAngle(me->GetVictim()));
             me->SetTarget(ObjectGuid::Empty);
             scheduler.DelayAll(6999ms);
-            scheduler.Schedule(7s, [this](TaskContext /*context*/)
+            scheduler.Schedule(7s, [this](TaskContext)
             {
                 _castingQuake = false;
                 me->SetReactState(REACT_AGGRESSIVE);
                 me->GetMotionMaster()->MoveChase(me->GetVictim());
-                DoCastSelf(SPELL_BLAST_NOVA);
+            });
+            context.Repeat(56300ms, 64300ms);
+        }).Schedule(55650ms, [this](TaskContext context)
+        {
+            DoCastSelf(SPELL_BLAST_NOVA);
+            scheduler.DelayAll(9999ms);
```

Comment:
was this sniffed? because it looks very weird

Reply:
Just 1ms below 10s. Following what I've seen happen on Gruul. That's the channel time of blast wave

Reply:
> was this sniffed? because it looks very weird

But I can make it 10s if you want?